Grand Circus - OOP/GrandCircus/CircusModel/Bear.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal class Bear : AnimalBase, IAnimal
+    {
+        public Bear(string name) : base (name)
+        {
+            base.SpeciesName = "bear";
```

Comment:
SpeciesName must not be set outside AnimalBase class. Hint: AnimalBase must receive 2 parameters and the second one is set from here (applies to all animal classes).